client/lib/presales-chat/index.ts

```diff
@@ -65,3 +71,17 @@ export function usePresalesChat( keyType: KeyType, enabled = true, skipAvailabil
 		isPresalesChatAvailable,
 	};
 }
+
+// This function consolidates the options that are passed to usePresalesChat for better readability and backward compatibility.
+export function usePresalesChatWithOptions(
+	keyType: KeyType,
+	options: {
+		enabled?: boolean;
+		skipAvailabilityCheck?: boolean;
+		skipEligibilityCheck?: boolean;
+	} = {}
+) {
+	const { enabled = true, skipAvailabilityCheck = false, skipEligibilityCheck = false } = options;
+
+	return usePresalesChat( keyType, enabled, skipAvailabilityCheck, skipEligibilityCheck );
+}
```

Comment:
I really like this alternate version of the hook -- it feels much cleaner to write. Eventually (IMO) it'd be super cool if we made it the default and phased out the original.